x-pack/filebeat/module/snyk/vulnerabilities/config/config.yml

```diff
@@ -90,13 +90,6 @@ processors:
   - fingerprint:
       fields: ["json.issue.id"]
       target_field: "@metadata._id"
-  - script:
```

Comment:
@andrewkroh Will this break things? I assume as part of the packages you already have a workaround here?

Reply:
@P1llus ^